### PR TITLE
update yari-content-ko team

### PR DESCRIPTION
**Pull request by hochan222**
### Description

update yari-content-ko team, Welcome @1ilsang!

[docs/ko/guides/organizer-guide.md](https://github.com/mdn/translated-content/pull/18413/files#diff-d3a83fc3620021ae19dfd5b732fd9b593e6b7212b122c673e5fddef45448fd5b) file for @yujo11 @GwangYeol-Im 

### Motivation

We welcomed new team member in the issue below. Requesting a documentation update on this.

### Related issues and pull requests

- https://github.com/mdn/translated-content/issues/18056
- https://github.com/orgs/mdn/teams/yari-content-ko/
- 2023: https://github.com/mdn/translated-content/pull/12430

### CODE REVIEW

**Review by Graywolf9 — APPROVED** [review]
Welcome to the new members! 

LGTM for `es`

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ slug: MDN/Community/Contributing/Translated_content
 ### 韩语（ko）
 
 - 讨论组：[Discord (#korean channel)](/discord)、[Kakao Talk (#MDN Korea)](https://open.kakao.com/o/gdfG288c)
-- 目前的志愿者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[cos18](https://github.com/cos18)、[GwangYeol-Im](https://github.com/GwangYeol-Im)、[pje1740](https://github.com/pje1740)、[yujo11](https://github.com/yujo11)、[wisedog](https://github.com/wisedog)、[swimjiy](https://github.com/swimjiy)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)
+- 目前的志愿者：[hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [pje1740](https://github.com/pje1740), [wisedog](https://github.com/wisedog), [swimjiy](https://github.com/swimjiy), [jho2301](https://github.com/jho2301), [sunhpark42](https://github.com/sunhpark42), [1ilsang](https://github.com/1ilsang)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:43]:
```suggestion
- 目前的志愿者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[cos18](https://github.com/cos18)、[pje1740](https://github.com/pje1740)、[wisedog](https://github.com/wisedog)、[swimjiy](https://github.com/swimjiy)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
```

**Review by yin1999 — COMMENTED** [review]
For zh-CN: we use `、` rather than comma (`,`) to separate items :)

**Review by yin1999 — APPROVED** [review]
LGTM for `l10n-zh` :)

**Comment by mfuji09** [comment]
LGTM for `ja`!

**Review by leon-win — APPROVED** [review]

**Comment by leon-win** [comment]
Welcome to the MDN, @1ilsang! :hugs:

**Review by cw118 — APPROVED** [review]
Looks good for l10n-fr, welcome to MDN 🎉

**Comment by dipikabh** [comment]
Hi everyone, looks like this PR is ready to be merged. Do let us know if any support is required.

Thanks a lot for all your contributions 🙏 

Welcome to the team, @1ilsang 🎉

**Comment by hochan222** [comment]
Thank you everyone. 

Have a good weekend :)

**Pull request merged**